<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>669</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Range of Motion Tracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #f9f9f9;
    font-size: 11pt;
    font-family: &quot;Segoe UI&quot;, &quot;Helvetica Neue&quot;, Arial;
	color: #000
}

QGroupBox {
    font-weight: bold;
    border: 1px solid gray;
    border-radius: 6px;
    margin-top: 10px;
    padding: 6px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 5px;
}

QLineEdit, QDateEdit, QSpinBox {
    border: 1px solid #ccc;
    border-radius: 4px;
    padding: 4px;
    background-color: #ffffff;
	color: #000
}
QLineEdit:focus, QDateEdit:focus, QSpinBox:focus {
    border: 1px solid #2d89ef;
    outline: none;
}

QPushButton {
    background-color: #2d89ef;
    color: white;
    border-radius: 6px;
    padding: 6px 12px;
}
QPushButton:hover {
    background-color: #1a5fb4;
}
QPushButton:pressed {
    background-color: #104a85;
}

#labelROMValue {
    font-size: 16pt;
    font-weight: bold;
    color: #333333;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,1">
    <item>
     <widget class="QGroupBox" name="groupBoxPatientInfo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Patient Info</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelDate">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="displayFormat">
          <string>MM/dd/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxRangeOfMotionInput">
      <property name="title">
       <string>Range of Motion Input</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="labelJoint">
         <property name="text">
          <string>Joint:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxJointSelect">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <item>
          <property name="text">
           <string>Knee</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hip</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="labelStartAngle">
         <property name="text">
          <string>Start Angle:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBoxStartAngle">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="suffix">
          <string> °</string>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="labelEndAngle">
         <property name="text">
          <string>End Angle:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBoxEndAngle">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="suffix">
          <string> °</string>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButtonSaveMeasurement">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Save Measurement</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButtonClear">
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxResults">
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidgetHistory">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Joint</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Start</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>End</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ROM</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelROMText">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ROM (degrees):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="labelROMValue">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0 °</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
